socorro/app/socorro_app.py

```diff
@@ -202,6 +215,7 @@ def fix_exit_code(code):
                     respond_to_SIGHUP,
                     logger=config.logger
                 )
+                spit_out_version_json(config.logger)
```

Comment:
It helps me if the version information is logged before all the config information. I'd move this line above the `try` block. Can you do that, please?

Reply:
@willkg Sure! I fixed it. Could you check it, please?